src/Compilers/CSharp/Portable/CSharpResources.resx

```diff
@@ -7929,4 +7929,22 @@ To remove the warning, you can use /reference instead (set the Embed Interop Typ
   <data name="ERR_InterceptsLocationDataInvalidPosition" xml:space="preserve">
     <value>The data argument to InterceptsLocationAttribute refers to an invalid position in file '{0}'.</value>
   </data>
+  <data name="ERR_PartialPropertyMissingImplementation" xml:space="preserve">
+    <value>Partial property '{0}' must have an implementation part.</value>
+  </data>
+  <data name="ERR_PartialPropertyMissingDefinition" xml:space="preserve">
+    <value>Partial property '{0}' must have an definition part.</value>
+  </data>
+  <data name="ERR_PartialPropertyDuplicateDefinition" xml:space="preserve">
+    <value>A partial property may not have multiple defining declarations</value>
+  </data>
+  <data name="ERR_PartialPropertyDuplicateImplementation" xml:space="preserve">
+    <value>A partial property may not have multiple implementing declarations</value>
```

Comment:
Should we consider mentioning something about not allowing auto-properties to be partial? Seems like that might be a common mistake, and a user might think the error is confusing without calling that out explicitly.